app.js

```diff
@@ -10,12 +11,23 @@ var foodsRouter = require('./routes/api/v1/foods');
 var mealsRouter = require('./routes/api/v1/meals');
 
 var app = express();
+app.use(cors());
+app.use((req, res, next) => {
+  res.header('Access-Control-Allow-Origin', '*');
```

Comment:
Just noting for future reference, we will need to change this to the django application once it is fully up and running